rustls/examples/internal/bogo_shim.rs

```diff
@@ -315,15 +315,15 @@ struct FixedSignatureSchemeClientCertResolver {
 impl client::ResolvesClientCert for FixedSignatureSchemeClientCertResolver {
     fn resolve(
         &self,
-        acceptable_issuers: &[&[u8]],
+        hint_ca_subjects: &[&[u8]],
```

Comment:
I think we should be consistent about root vs ca. Maybe this could be root_subject_hints?

Reply:
Alternatively, `root_hint_subjects` might align better with the `root_hints` from elsewhere? Or we could just use `root_hint_subjects` across the board maybe.

Reply:
I standardized on `root_hint_subjects` throughout.